### PR TITLE
Adding feature to control Android ID Fetch

### PR DESCRIPTION
**Pull request by sojanpr**
Adding feature to control android id read 
@aaustin @Sarkar

### CODE REVIEW

**Comment by Sarkar** [comment]
👍 awesome!

File: Branch-SDK-TestBed/AndroidManifest.xml [hunk 1]
```diff
@@ -58,6 +58,7 @@
             <meta-data android:name="io.branch.sdk.auto_link_request_code" android:value="@integer/AutoDeeplinkRequestCode" />
         </activity>
 
+        <meta-data android:name="io.branch.sdk.DisableDeviceIDCapture" android:value="false" /> <!-- Set to true to disable Android ID capture-->
```
Comment (aaustin) [inline Branch-SDK-TestBed/AndroidManifest.xml]: I don't like the idea of putting more and more configuration in the manifest. Can we make this a part of the Application class?

**Comment by sojanpr** [comment]
Changed from Manifest con to a class method. Ready for review

**Comment by Sarkar** [comment]
solid 👍

**Comment by aaustin** [comment]
coolies 👍 sorry for being a stickler

**Pull request merged**